source/client-side-encryption/client-side-encryption.rst

```diff
@@ -914,6 +994,12 @@ ClientEncryption
    class ClientEncryption {
       ClientEncryption(opts: ClientEncryptionOpts);
 
+      // The "Create Encrypted Collection" helper is a convenience function wrapping CreateCollection. It will
+      // create a collection with encrypted fields, automatically allocating and assigning new data encryption
+      // keys. It returns a handle to the  new collection, as well as a document consisting of the generated
+      // "create" options. Refer to "Create Encrypted Collection Helper"
```

Comment:
```suggestion
      // "encryptedFields" options. Refer to "Create Encrypted Collection Helper"
```

I expect some drivers may not have a native way to convert CreateCollectionOptions to BSON. Only returning "encryptedFields" is needed to provide the datakeys to the caller.